spec/unit/installer/xcode/multi_pods_project_generator_spec.rb

```diff
@@ -89,8 +89,8 @@ class Xcode
 
             @installation_options = Pod::Installer::InstallationOptions.new
 
-            @generator = MultiPodsProjectGenerator.new(config.sandbox, aggregate_targets, pod_targets, @analysis_result,
-                                                       @installation_options, config)
+            @generator = MultiPodsProjectGenerator.new(config.sandbox, aggregate_targets, pod_targets, @analysis_result.all_user_build_configurations,
+                                                       @installation_options, config, nil)
```

Comment:
isnt `nil` a default value ? I do not remember

Reply:
That's actually the `project_object_version`. Shouldn't really have a default value, this was a good use in the test. Will update with an appropriate value.